src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
Are you prepared to handle this file in the package? What happens if I don't have this file?